manifests/pool.pp

```diff
@@ -3,6 +3,7 @@
 define dhcp::pool (
   Stdlib::IP::Address::V4 $network,
   Stdlib::IP::Address::V4 $mask,
+  Stdlib::IP::Address::V4 $host_mask        = $mask,
```

Comment:
Although documentation is missing for the rest of the parameters could you add puppet-strings style docs for the new param?